File: lcm.py
from gcf import greatest_common_factor
def least_common_multiple(a,b):
  return int((a * b) / greatest_common_factor(a,b))


def main(args):
    args = args[1:]
    if(len(args) == 0):
        print("Missing a and b! Try again with a and b provided!")
        return
    elif(len(args) == 1):
        print("Missing b value! Try again!")
        return
    print(f"Least common multiple between {args[0]} and {args[1]} is: {least_common_multiple(int(args[0]), int(args[1]))}")

File: c/lcm.c
#include <stdio.h>
#include <stdlib.h>

#include "gcf.h"

int least_common_multiple(int a, int b)
{
  return (int)((a * b) / greatest_common_factor(a, b));
}

int run_lcm(int argc, char *argv[])
{
  if (argc < 4)
  {
    printf("Missing parameters a and b, try again!");
    return 1;
  }
  printf("Least common multiple between %d and %d is %d\n", atoi(argv[2]), atoi(argv[3]), least_common_multiple(atoi(argv[2]), atoi(argv[3])));
  return 0;
}